packages/rest-typings/src/apps/index.ts

```diff
@@ -96,9 +96,9 @@ export type AppsEndpoints = {
 		};
 	};
 
-	'/apps/featured': {
+	'/apps/featured-apps': {
```

Comment:
Isn't this a breaking change?

Reply:
No sir. That route was added in https://github.com/RocketChat/Rocket.Chat/pull/26416 and was added specifically for this new feature. The Marketplace API doesn't have the old route after we tested in staging and realized it wouldn't work.